pom.xml

```diff
@@ -151,24 +151,24 @@ THE SOFTWARE.
       <dependency>
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
-        <version>4.11</version>
+        <version>4.12</version>
```

Comment:
@KostyaSha Actually I did (well, `-Plight-test`), which is why there are other changes. Note the PR build's 0 test failures.